packages/indexer-agent/src/agent.ts

```diff
@@ -780,6 +788,7 @@ class Agent {
     )
   }
 
+  // TODO: Skip evaluation or short-circuit if item already in queue or AllocationManagementMode = MANUAL
```

Comment:
Open to ideas here!

Reply:
The problem with filtering targetAllocations is that the agent will then think they should be removed if there is an activeAllocation not in the targetAllocation list.  
What I've done now is add a check in the reconciliation loop for approved actions and it will wait until the next reconciliation loop (effectively pause) until those actions are executed.